modules/redis/scanner.go

```diff
@@ -69,26 +74,34 @@ type Result struct {
 	// required error even if --password is provided.
 	PingResponse string `json:"ping_response,omitempty"`
 
+	// AuthResponse is only included if --password is set.
+	AuthResponse string `json:"auth_response,omitempty"`
+
 	// InfoResponse is the response from the INFO command: "Lines can contain a
 	// section name (starting with a # character) or a property. All the
 	// properties are in the form of field:value terminated by \r\n."
 	InfoResponse string `json:"info_response,omitempty"`
 
-	// QuitResponse is the response from the QUIT command -- should be the
-	// simple string "OK" even when authentication is required, unless the
-	// QUIT command was renamed.
-	QuitResponse string `json:"quit_response,omitempty"`
+	// Version is read from the InfoResponse (the field "server_version"), if
+	// present.
+	Version string `json:"version,omitempty"`
+
+	// OS is read from the InfoResponse (the field "os"), if present. It specifies
+	// the OS the redis server is running.
+	OS string `json:"os,omitempty"`
```

Comment:
Good catch - any other common / useful fields in there?

Reply:
Another field that might be useful is `redis_mode`, which specifies what mode Redis is running on, whether that be `cluster` or `standalone`. However, other than that I'm not sure that there's much more that might be useful

Reply:
That could be nice to have -- especially since we're already reading it.

Reply:
We now collect the `redis_mode`